src/pages/todoSetup/TodoSetup.js

```diff
@@ -0,0 +1,86 @@
+import React, {useState, useEffect} from 'react';
+
+// Components
+import TodoForm from '../../molecules/todoForm';
+import TodoList from '../../molecules/todoList';
+import Header from '../../atoms/header';
+
+// Constants
+import { MOCK_TODO_LIST } from '../../constants/mockData';
+
+// utils
+import { fetchData } from '../../utils/fetchData';
+
+// Styles
+import "./TodoSetup.css"
+
+const TodoSetup = () => {
+    const [todoVal, setTodoVal] = useState("");
+    const [todoList, setTodoList] = useState([]);
```

Comment:
We can instead make a constant at root level constants-> todo.general.js and it will contain : 

EMPTY_ARRAY = []
EMPTY_OBJECT= {}
and export it